src/main/java/seedu/progresschecker/ui/UiManager.java

```diff
@@ -30,7 +30,7 @@
     public static final String FILE_OPS_ERROR_DIALOG_CONTENT_MESSAGE = "Could not save data to file";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/progress_checker_32.png";
```

Comment:
Might want to change this back to ```address_book_32.png``` for the image file to be valid. P.S. Not recommended to rename the image file to solve this instead because the image does not belong to us so we shouldn't claim it.